.github/workflows/ci.yml

```diff
@@ -105,7 +106,7 @@ jobs:
       - name: install_rust
         uses: actions-rs/toolchain@v1
         with:
-          toolchain: stable
+          toolchain: nightly
```

Comment:
Can I assume the change in clippy toolchain is because of a configuration setting which is not yet on the stable channel? I would prefer to stay on stable, as there have been some suggestions in the past, added to the default category, which would have been more reasonable in the pendantic category. Since builds will fail when new clippy versions are released (this is a good thing in my opinion), which requires extra attention, I would like to keep clippy changes which may not be that important to a minimum.